projects/planner/src/app/diver/diver.component.spec.ts

```diff
@@ -29,60 +30,70 @@ describe('DiverComponent', () => {
         unitConversion = TestBed.inject(UnitConversion);
         component.diver = new DiverOptions();
         fixture.detectChanges();
+        pageObject = new DiverComponentPageObject(fixture);
     });
 
     it('should change rmv and stressRmv values and emit', () => {
-        expect(component.diver.rmv).toBe(20);
-        expect(component.diver.stressRmv).toBe(30);
+        expect(pageObject.rmv).toBe(20);
+        expect(pageObject.stressRmv).toBe(30);
 
         const newRmv = 15;
         const newStressRmv = 25;
+        pageObject.setRmvAndStressRmv(newRmv, newStressRmv);
 
-        component.diverForm.setValue({ rmv: newRmv, stressRmv: newStressRmv });
+        expect(pageObject.rmv).toBe(newRmv);
+        expect(pageObject.stressRmv).toBe(newStressRmv);
+        expect(pageObject.changed).toBeTrue();
+    });
 
-        let wasEmitted = false;
-        component.changed.subscribe(() => wasEmitted = true);
+    class DiverComponentPageObject {
+        constructor(private componentFixture: ComponentFixture<DiverComponent>) {}
 
-        component.inputChanged();
 
-        expect(component.diver.rmv).toBe(unitConversion.toLiter(newRmv));
-        expect(component.diver.stressRmv).toBe(unitConversion.toLiter(newStressRmv));
-        expect(wasEmitted).toBeTrue();
-    });
+        get changed(): boolean {
+            let wasEmitted = false;
+            this.componentFixture.componentInstance.changed.subscribe(() => wasEmitted = true);
+            return wasEmitted;
+        }
+
+        get rmv(): number {
+            return this.componentFixture.componentInstance.diver.rmv;
```

Comment:
Take the value from the page input element not from the component it self.
The same for all other properties